Dockerfile

```diff
@@ -44,9 +45,9 @@ ENV APP_PORT=49994
 EXPOSE $APP_PORT
 
 WORKDIR /
-COPY --from=builder /go/src/github.com/edgexfoundry/device-gpio/LICENSE /
-COPY --from=builder /go/src/github.com/edgexfoundry/device-gpio/Attribution.txt /
-COPY --from=builder /go/src/github.com/edgexfoundry/device-gpio/cmd/ /
+COPY --from=builder /device-gpio/LICENSE /
+COPY --from=builder /device-gpio/Attribution.txt /
+COPY --from=builder /device-gpio/cmd/ /
 
 ENTRYPOINT ["/device-gpio"]
-CMD ["-cp=consul://edgex-core-consul:8500", "--registry", "--confdir=/res"]
+CMD ["-cp=consul.http://edgex-core-consul:8500", "--registry", "--confdir=/res"]
```

Comment:
Should stay as just "consul"

Reply:
This was @lenny-intel's suggestion https://github.com/edgexfoundry/device-gpio/pull/3#discussion_r653071433

![image](https://user-images.githubusercontent.com/1890273/122427391-ee249500-cf45-11eb-8676-cf204db96940.png)

Reply:
@iain-anderson @lenny-intel  I can go either way. Let me know what this value should be.

Reply:
Don't mind, @lenny-intel suggested to not use the http in another context.

Reply:
@iain-anderson , specifying it in the Type field in TOML is non-standard (not supported in Go services) and this is for Configuration Provider, not Registry. More of a consistency thing as all App/core/support and most Device services have it this way.